cropper/src/main/java/com/canhub/cropper/BitmapUtils.kt

```diff
@@ -428,15 +425,11 @@ internal object BitmapUtils {
         customOutputUri: Uri?,
     ): Uri? {
         val newUri = customOutputUri ?: buildUri(context, compressFormat)
-        var outputStream: OutputStream? = null
-        try {
-            outputStream = context.contentResolver.openOutputStream(newUri!!, WRITE_AND_TRUNCATE)
 
-            bitmap.compress(compressFormat, compressQuality, outputStream)
-        } finally {
-            closeSafe(outputStream)
+        return context.contentResolver.openOutputStream(newUri!!, WRITE_AND_TRUNCATE).use {
+            bitmap.compress(compressFormat, compressQuality, it)
+            newUri
```

Comment:
What happens here if `newUri` is null or the `compress` method fails?

Reply:
https://github.com/CanHub/Android-Image-Cropper/pull/440#discussion_r979828351